Controller/Notification/Index.php

```diff
@@ -103,31 +105,39 @@ public function __construct(
     }
 
     /**
+     * @throws InputException
      * @throws InvalidExternalIdException
-     * @throws InvalidSignatureException
+     * @throws NoSuchEntityException
      */
     public function execute()
     {
         $jsonContent = $this->request->getContent();
         $content = json_decode($jsonContent, true);
-        $application = $this->getApplication($content['externalId'])
-                            ->getData();
 
-        if (empty($application)) {
-            throw new InvalidExternalIdException('Invalid external ID!');
-        }
+        if (isset($content['externalId'])) {
```

Comment:
niepotrzebnie wcięcie powstało chowające istotę problemu. Tutaj chcesz podkreślić, że jeśli nie ma `externalId`, to nie jest to błąd i metoda powinna się zakończyć bez rzucania wyjątku.
Lepiej wtedy napisać 
```
if (!isset($content['externalId'])) {
    return;
}
```
Łatwiej wtedy znaleźć przyczynę błędu, jeśli okaże się, że brak `externalId` błędem jednak jest. Dodatkowo jedno wcięcie mniej. Która wersja wygląda lepiej? Mogę rozwinąć przykład o kilka pól więcej.


```
$v = ['a' => 1, 'b' => 2, 'c' => 3];
if (isset($v['a'])) {
    if (isset($v['b'])) {
        if (isset($v['c'])) {
           // do stuff
        }
    }
}
```

```
$v = ['a' => 1, 'b' => 2, 'c' => 3];
if (!isset($v['a'])) {
    return;
}
if (!isset($v['b'])) {
    return;
}
if (!isset($v['c'])) {
    // oh, this looks like error
}
// do stuff
```